Cargo.toml

```diff
@@ -21,18 +21,20 @@ path = "examples/embark.rs"
 [dependencies]
 base64 = "0.21"
 http = "0.2"
-jsonwebtoken = "8.1"
+# While waiting for https://github.com/Keats/jsonwebtoken/pull/303 to get in we need
+# to rely on this forked commit for the `pem` crate to use `base64 0.21`
+jsonwebtoken = { git = "https://github.com/EmbarkStudios/jsonwebtoken", rev = "ccd4394b49c1b92590ba6cca0f5d1ec44c813acd" }
```

Comment:
not possible to have git dependencies in crates published on crates.io. have to get this released before so one can reference a published version of the dependency here

Reply:
> Though we should wait for https://github.com/Keats/jsonwebtoken/pull/303 to get in
> and I feel a little reluctant to release a new version of tame-oidc before that PR gets in.

That makes the decision even easier.